src/Debug.tsx

```diff
@@ -12,14 +12,16 @@ export default function Debug({query, params}: {query: string; params: {[key: st
           <Code>{query}</Code>
         </Box>
       </Stack>
-      <Stack space={4}>
-        <Box>
-          <Label>Params</Label>
-        </Box>
-        <Box>
-          <Code>{JSON.stringify(params)}</Code>
-        </Box>
-      </Stack>
+      {params && (
```

Comment:
`params` can now be null if they can't be resolved

Reply:
I wonder about this though - maybe it should just be `undefined` if not resolved?